Tools/autotest/autotest.py

```diff
@@ -1131,6 +1131,9 @@ def format_epilog(self, formatter):
 
     try:
         if not run_tests(steps_to_run):
+            if os.environ.get("COVERAGE", False):
+                # Don't report failure on coverage test
+                sys.exit(0)
```

Comment:
Why not?

Reply:
the coverage run is around 2h30 currently, and there can be too much source of failure. So I prefer it to run fully even if we miss to complete some test, specially examples that can fails due to issue with CI.

Reply:
I think my point here was that we should ignore the error in the caller to `autotest.py`, not change the return value here.